app/views/shared/dialogs/_dialog_field_drop_down_list.html.haml

```diff
@@ -19,7 +24,12 @@
   - if field.auto_refresh
     :javascript
       dialogFieldRefresh.listenForAutoRefreshMessages("#{field.id}", function() {
-        var selectedValue = $('select[name="#{field.name}"]').val();
+        var field = $('select[name="#{field.name}"]').length;
+        if (field) {
+          var selectedValue = $('select[name="#{field.name}"]').val();
+        } else {
+          var selectedValue = $('select[name="#{field.name}[]"]').val();
+        }
```

Comment:
yeah, it doesn't work, the field name is "#{field.name}[]", without the [] suffix, it doesn't find it, cause you are searching for a different name